ssl/statem/statem_lib.c

```diff
@@ -2049,10 +2049,11 @@ int ssl_set_version_bound(int method_version, int version, int *bound)
 
     valid_tls = version >= SSL3_VERSION && version <= TLS_MAX_VERSION_INTERNAL;
     valid_dtls =
-        DTLS_VERSION_LE(version, DTLS_MAX_VERSION_INTERNAL) &&
-        DTLS_VERSION_GE(version, DTLS1_BAD_VER);
+        (version == DTLS1_BAD_VER)
```

Comment:
Is this correct?  If it is, there should be a comment explaining what this is a valid DTLS version.

Reply:
Yes, it is correct. Comment added.